vizro-core/examples/features/app.py

```diff
@@ -404,6 +409,9 @@
             column="size",
             selector=vm.Slider(title="Slider (Tips - size)", step=1, value=2),
         ),
+        vm.Filter(
```

Comment:
Thank you! I forgot to test the example after updating the file 😄. I believe this error is valid since `year' in this case is a datetime type column.

I don't think we want to allow our users to explicitly set any selector type if they want (regardless of column type). Hence we put validations like this. Do you think we should? 

I updated the column for this filter to `lifeExp`. 😄

Reply:
No I think all good - just wanted to make you aware it didnt run